<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyFrameCapUI</class>
 <widget class="QDialog" name="KeyFrameCapUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="ImageLabel" name="SLeftWin">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
  <widget class="ImageLabel" name="SRigWin">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
  <widget class="QListWidget" name="SListSeq">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>1331</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LOK">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>500</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/ok-button.jpg</normaloff>Resources/ok-button.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ROK">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>490</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/ok-button.jpg</normaloff>Resources/ok-button.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="ImageLabel" name="SKey1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
  <widget class="ImageLabel" name="SKey2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
  <widget class="ImageLabel" name="SKey3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
  <widget class="ImageLabel" name="SKey4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>600</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
  <widget class="ImageLabel" name="SKey5">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>590</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Input video</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header location="global">ImageShowWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
